### DISCUSSION TITLE
Total record count

### DISCUSSION THREAD

**Discussion by wolfcrister** [rappen/FetchXMLBuilder#431] [Ideas]
Hey! It worked! (Seems like you have to verify your email address. Weird stuff).

So let's try it here then.

Could we have a counter for total records that passes 5000? Like the one in "Bulk workflow executioner"?
Or is it one already that I haven't found?

![image](https://user-images.githubusercontent.com/75740455/101643895-e20ac880-3a34-11eb-828f-f61addb0f0d9.png)

**Comment by BetimBeja** [comment]
You mean like [Fast Record Counter](https://www.xrmtoolbox.com/plugins/Fic.XTB.FastRecordCounter/)

**Reply by wolfcrister** [reply]
You really released that yesterday? Odd coincidence.
But yeah, that one's really good, although I would like that functionality in FetchXML builder so I can count within my filters =)

**Reply by rappen** [reply]
Actually @DynamicsNinja released it...!
See https://github.com/DynamicsNinja/FastRecordCounter
